packages/@o3r/schematics/src/utility/loaders.ts

```diff
@@ -76,13 +76,23 @@ export function readPackageJson(tree: Tree, workspaceProject: WorkspaceProject)
 
 /**
  * Return the type of install to run depending on the project type (Peer or default)
+ * @deprecated use {@link getProjectNewDependenciesTypes instead}, will be removed in V11
  * @param project
  * @param tree
```

Comment:
no tree and v12?

Reply:
nop, will not wait 1 year to remove a schematic helper